Lib/test/test_signal.py

```diff
@@ -608,6 +608,15 @@ def test_itimer_prof(self):
         # and the handler should have been called
         self.assertEqual(self.hndl_called, True)
 
+    def test_setitimer_tiny(self):
+        # bpo-30807: C setitimer() takes a microsecond-resolution interval.
+        # Check that float -> timeval conversion doesn't round
+        # the interval down to zero, which would disable the timer.
+        self.itimer = signal.ITIMER_REAL
+        signal.setitimer(self.itimer, 1e-6)
```

Comment:
You don't understand. `1e-6` is the minimum advertised resolution for `setitimer`, that's why it's supposed to work.